src/http.ts

```diff
@@ -153,7 +151,7 @@ export class LemmyHttp {
   #apiUrl: string;
   #headers: { [key: string]: string } = {};
   #pictrsUrl: string;
-  #fetchFunction = fetch;
+  #fetchFunction = fetch.bind(globalThis);
```

Comment:
Is there a point to still assigning to this private field instead of using `fetch` directly in `#wrapper`?

Reply:
fetch is one of the options you can pass in. Voyager takes advantage of this.